### PR TITLE
Issue/15/sign up view

### PR DESCRIPTION
**Pull request by Bedpaw**
To check

### CODE REVIEW

File: src/components/organisms/form/sign-up.vue [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    <a-input
+      v-model="repeatedPassword"
+      placeholder="powtórz hasło"
+      type="password"
+      :disabled="blockForm"
+    />
+    <a-input
+      v-model="values.patrolName"
+      placeholder="nazwa patrolu"
+      type="email"
```
Comment (Drego31) [inline src/components/organisms/form/sign-up.vue]: `type="email"` ? Should it be here?

File: src/components/organisms/form/sign-up.vue [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      :disabled="blockForm"
+    />
+    <a-input
+      v-model="values.eventCode"
+      placeholder="kod wydarzenia"
+      type="email"
```
Comment (Drego31) [inline src/components/organisms/form/sign-up.vue]: `type="email"` ? Should it be here?

File: src/components/organisms/form/sign-up.vue [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        this.message = text
+        setTimeout(() => {
+          this.message = ''
+          this.blockForm = false
+          resolve()
+        }, 2000)
+      })
+    },
+    onSignUp () {
+      this.setMessage('Konto utoworzono pomyślnie - kliknij w link aktywacyjny na swojej skrzynce mailowej')
```
Comment (Drego31) [inline src/components/organisms/form/sign-up.vue]: This message is too long to be inside button.
It should be show in separated message block.

Comment (Drego31) [reply src/components/organisms/form/sign-up.vue]: And time to read this message is too short.

File: src/components/organisms/form/sign-up.vue [hunk 4]
```diff
@@ -0,0 +1,112 @@
+<template>
```
Comment (Drego31) [inline src/components/organisms/form/sign-up.vue]: Here is a lot of components to refactor (duplicates - comparison with `o-form-sign-in`).

File: src/components/organisms/form/sign-up.vue [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+  <div class="f-pb-1">
+    <a-input
```
Comment (Drego31) [inline src/components/organisms/form/sign-up.vue]: You can make `v-for` in `a-input` - duplicates.

File: src/components/organisms/form/sign-up.vue (hunk 1 above)
Comment (Bedpaw) [reply src/components/organisms/form/sign-up.vue]: corrected

File: src/components/organisms/form/sign-up.vue (hunk 2 above)
Comment (Bedpaw) [reply src/components/organisms/form/sign-up.vue]: corrected

File: src/components/organisms/form/sign-up.vue (hunk 4 above)
Comment (Bedpaw) [reply src/components/organisms/form/sign-up.vue]: created mixins for these components and a-button-submit

File: src/components/organisms/form/sign-up.vue (hunk 3 above)
Comment (Bedpaw) [reply src/components/organisms/form/sign-up.vue]: I've separated message to other block.

**Comment by Drego31** [comment]
You can merge @Bedpaw

**Pull request merged**